odoo/addons/base/models/res_users.py

```diff
@@ -1401,15 +1401,22 @@ def _update_user_groups_view(self):
                 else:
                     # application separator with boolean fields
                     app_name = app.name or 'Other'
-                    xml4.append(E.separator(string=app_name, colspan="4", **attrs))
+                    xml4.append(E.separator(string=app_name, **{'class': 'mt-3'}, **attrs))
```

Comment:
I think that we should add this margin-top on 'o_horizontal_separator'. We would then make a separate commit to fix this more globally.

Reply:
https://github.com/odoo/odoo/pull/102900